### PR TITLE
fix post stacking order, address #698

### LINKED ISSUE #698: Order of items in queue changes after share happens
@irinelenache Can you help me confirm this?

Connect say...3 test accounts to the plugin. Create say 5 test posts. Click start sharing, **after this first share happens** and shares are sent to the 3 accounts, go to the **Share Queue**. Make note of all the **upcoming share items**.

When the **next share** comes around, check the log to see if the posts shared were the actual posts that had to be shared according to the previously queued items(that you marked down) as well as to the respective account the queue card listed. 

Then, go back to the **Share Queue** area and see if the next batch of scheduled queue items corresponds with what you previously marked down as well. To keep track easily name the posts as "Post 1", "Post 2", "Post 3" etc etc. 

How it should be working, say **after the initial share** happens the queue looks like this:

- Post 2 
- Post 1 
- Post 3 
- Post 6
- Post 5
- Post 4
- Post 7
- Post 8
- Post 9

If you check the logs after the **next set of shares happen**, then the posts that should have gone out are:

Post 1,2,3 (order doesn't matter here)

When you go back to the sharing queue, then the order of the queue should now be:

- Post 6
- Post 5
- Post 4
- Post 7
- Post 8
- Post 9
...
...
...

The accounts it's set to share to will also be the same as what you initially took down. Please reach out on Slack if you need some more explanation

Use the current version of ROP from the wp.org repo to test this

### PR DESCRIPTION
**Pull request by UVLabs**
fixes #698

### CODE REVIEW

**Comment by UVLabs** [comment]
@irinelenache This pull contains a fix for #698 the next posts to share should now be added to the bottom of the queue instead of the top.

I don't want to merge it into v9.0.0 yet since it affects the operation of the sharing queue. Can you please test and see if you find any issues regarding sharing/sharing queue?

Be sure to test all features related to the sharing queue.

I have attached a zip of the plugin

[tweet-old-post.zip](https://github.com/Codeinwp/tweet-old-post/files/6260706/tweet-old-post.zip)

**Comment by irinelenache** [comment]
@UVLabs Tested it with 4 social medias added, this is the output:
This was the queue before the first share:  https://user-images.githubusercontent.com/67897586/115023477-87dcb100-9ec7-11eb-8930-abce8cbade4e.png
These werer the shared posts: https://prnt.sc/11ikl5y
This was the queue after the share:  https://user-images.githubusercontent.com/67897586/115023564-a3e05280-9ec7-11eb-9c71-844eea6f9adc.png

If it worked on your end, send me again the versions of ROP & ROP Pro and i will try again

**Comment by UVLabs** [comment]
@irinelenache start monitoring it **after** the first share, once that first share cycle happens the sharing queue should hopefully normalize and shared posts should get cut off the top and new ones added to the bottom of the share queue.

I'm attaching another copy, the change is only in the lite version
[tweet-old-post.zip](https://github.com/Codeinwp/tweet-old-post/files/6325343/tweet-old-post.zip)

**Comment by irinelenache** [comment]
@UVLabs Tested it again, and after the first share it looks like it works fine. Local cron made it pretty hard to follow, and in the schedule, the medias don't appear always in the same order even though the correct posts are shared.
For example, if i added 3 social medias (facebook, twittter, linkedin) and 3 posts (post 1, post 2, post 3), the schedule appears to be:
First share (facebook - post 1, twitter - post 2, linkedin - post 3), Second share (facebook - post 2, twitter - post 3, linkedin - post 1), ...
After one share, the scheduled posts are the same, but the medias have another order in the schedule, something like:
linkedin - post 1, facebook - post 2, twitter - post 3

Basically this is not an issue, but it's a bit confusing when you to watch the schedule.

**Comment by UVLabs** [comment]
Thank you @irinelenache
